src/pybind11/PyBindImath/PyBindImathFrustum.cpp

```diff
@@ -10,20 +10,20 @@
 
 namespace PyBindImath {
 
-template <class T, class V, class M>
+template <class T, class V, class S>
 void register_frustum(pybind11::module& m, const char *name)
 {
     pybind11::class_<T> c(m, name);
     c.def(pybind11::init<>(), "Uninitialized by default")
     .def(pybind11::init<T>(), pybind11::arg("frustum"), "Copy constructor")
-    .def(pybind11::init<S>(), pybind11::arg("nearPlane"), pybind11::arg("farPlane"), pybind11::arg("fovx"), pybind11::arg("aspect"), "Initialize with basic frustum properties")
+    .def(pybind11::init<const S, S, S, S>(), pybind11::arg("nearPlane"), pybind11::arg("farPlane"), pybind11::arg("fovx"), pybind11::arg("aspect"), "Initialize with basic frustum properties")
```

Comment:
nearPlane, farPlane, fovx, and aspect are float/double, but S in this context is M44f/M44d, right?